subprojects/docs/src/main/resources/header.html

```diff
@@ -300,7 +300,7 @@ <h3 id="optimizing-build-performance">Optimizing Build Times</h3>
                 </ul>
             </li>
             <li><a href="../userguide/configuration_cache.html">Using the Configuration Cache</a></li>
-            <li><a href="https://scans.gradle.com/">Inspecting Gradle Builds</a></li>
+            <li><a href="../userguide/inspect.html">Inspecting Gradle Builds</a></li>
```

Comment:
Maybe move this to just below "Improving Performance ..."?

Reply:
Since this is just a cherry-pick, I'll add this in a future PR. In the initial PR, I didn't want to mess around with the sidebar ordering since people might have muscle memory built up :)